Source/Bugsnag/BugsnagSink.m

```diff
@@ -144,6 +144,16 @@ - (NSDictionary*) formatEvent:(BugsnagCrashReport*) report {
     [exception safeSetObject: report.errorClass forKey: @"errorClass"];
     [exception setObjectIfNotNil: report.errorMessage forKey: @"message"];
 
+    // HACK: For the Unity Notifier. We don't include ObjectiveC exceptions or threads
```

Comment:
This isnt a hack :-)

You could have notifier specific sink code that you call from here (so you call out to the notifier to make final adjustments to the report before its sent). This could then be used by users to solve some of the other ios issues we have.